src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/merchandise_return.yml

```diff
@@ -23,3 +23,31 @@ admin_merchandise_returns_filter:
     redirectRoute: admin_merchandise_returns_index
     _legacy_controller: AdminReturn
 #    _legacy_link: AdminReturn:submitFilterorder_return
+
+admin_merchandise_returns_edit:
+  path: /{merchandiseReturnId}/edit
+  methods: [GET, POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/CustomerService/MerchandiseReturn:edit
+    _legacy_controller: AdminReturn
+#    _legacy_link: AdminReturn:updateorder_return
+#    _legacy_parameters:
+#      id_order_return: merchandiseReturnId
+  requirements:
+    merchandiseReturnId: \d+
+
+admin_merchandise_returns_delete_product:
+  path: /{merchandiseReturnId}/products/{merchandiseReturnDetailId}/delete
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/CustomerService/MerchandiseReturn:deleteProduct
+    _legacy_controller: AdminReturn
+  #    _legacy_link: AdminReturn:deleteorder_return_detail
```

Comment:
Usually we wait for the 2 pages (listing and add/edit) to be merge before putting the `_legacy_link` as it hides the old page and redirects legacy URLs to Symfony URLs but since this PR migrates everything at once I think we can do enable them 😅 

Or maybe we'll do it in another PR, and remove legacy files at this time ?

Reply:
Maybe in another one, this allows to use the legacy one to compare both